### ISSUE TITLE
Did you tried it with mobiles ?

### ISSUE THREAD

**Issue by SeRgI1982** [tvjsx/trading-vue-js#226]
Hi,

I saw your chart on site: https://poocoin.app
Together with my friend we tried to find something similar for mobile (cross-platform Android/iOS) but I need to admit: your work is awesome and it is hard to find something similar for mobiles.
I wonder if I could re-use your chart lib in embedded webbrowser inside mobile app.

Did you have any experience with such approach ?

**Comment by adsonvinicius** [comment]
@SeRgI1982 Yes, this chart was tested in mobile. In tracking project for V 1.0 it was pointed this task.
